front/src/components/oauth2/OAuth2Config.jsx

```diff
@@ -0,0 +1,119 @@
+                        </p>
+                        <p>
+                          <MarkupText id={`integration.oauth2.instructions`} />
+                        </p>
+                        <div class="form-group">
+                          <div class="form-label">
+                            <Text id={`integration.oauth2.apiKeyLabel`} />
+                          </div>
+                          <div class="input-group">
+                            <Localizer>
+                              <input
+                                type="text"
+                                class="form-control"
+                                placeholder={<Text id={`integration.oauth2.clientId`} />}
+                                onInput={props.updateClientId}
+                                value={props.clientId}
+                              />
+                            </Localizer>
+                          </div>
+                        </div>
+                        <div class="form-group">
+                          <div class="input-group">
+                            <Localizer>
+                              <input
+                                type="text"
+                                class="form-control"
+                                placeholder={<Text id={`integration.oauth2.secretId`} />}
+                                onInput={props.updateSecretId}
+                                value={props.secretId}
+                              />
+                            </Localizer>
+                          </div>
+                        </div>
+                        <div class="form-group">
+                          <span class="input-group-append">
+                            <button class="btn btn-primary" onClick={props.startConnect}>
+                              <Text id={`integration.oauth2.buttonConnect`} />
+                            </button>
+                          </span>
+                        </div>
+                      </div>
+                    )}
+                    {props.oauth2GetStatus === RequestStatus.Success && props.clientIdInDb && (
+                      <div class="dimmer-content">
+                        <h2>
+                          <Text id={`integration.${integrationName}.settings.title`} />
+                        </h2>
+                        <div class="alert alert-info">
+                          <p>
+                            <Text id={`integration.${integrationName}.settings.complete`} />
+                          </p>
+                          <p>
+                            <b>
+                              <Text id={`integration.${integrationName}.settings.clientId`} />
+                            </b>
+                            {`${props.clientIdInDb}`}
+                          </p>
+                          <p>
+                            <Text id={`integration.${integrationName}.settings.afterIntegration`} />
```

Comment:
I'm not sure this key is set for all integrations.
Isn't it a OAuth client key?